db/migrate/20181222220236_add_description_to_api_secrets.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToApiSecrets < ActiveRecord::Migration[5.1]
+  def change
+    add_column :api_secrets, :description, :string
```

Comment:
Probably helpful to add a database validation to prevent null values.

```suggestion
    add_column :api_secrets, :description, :string, null: false
```

Reply:
It's done 😄 Since I was messing around with the migrations I went ahead and consolidated the migration files into a single migration.